system-test/busybench/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "busybench",
+  "version": "0.0.0",
```

Comment:
I switched to 1.0.0, because `npm init` automatically sets the version to 1.0.0.

The version here isn't really used (since this isn't an npm module).